bindings/java/src/main/native/Makefile.am

```diff
@@ -29,9 +29,10 @@ org_openucx_jucx_ucp_UcpListener.h:
 org_openucx_jucx_ucp_UcpConstants.h:
 org_openucx_jucx_ucp_UcpEndpoint.h:
 org_openucx_jucx_ucp_UcpWorker.h:
-org_openucx_jucx_ucs_UcsConstants.h:
-org_openucx_jucx_ucp_UcpContext.h:
-	$(MVNCMD) compile native:javah
+org_openucx_jucx_ucs_UcsConstants.h: $(topdir)/bindings/java/src/main/java/org/openucx/jucx/ucs/*.java
```

Comment:
maybe define a new variable NATIVE_HEADERS or JAVA_HEADERS
it will be part of : 1.BUILT_SOURCES 2. DISTCLEANFILES 3. Maven compile deps